src/components/Summary/DocumentsDelivery.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <section id="document-delivery-section">
+    <header>
+      <h2>1. Alteration Documents Delivery</h2>
+    </header>
+    <div class="pt-4">Copies of the alteration documents will be sent to the following email address listed below.</div>
+    <div class="pt-8 pr-8">
+      <v-container pb-6>
+        <v-row class="pl-4">
+          <v-col cols="2" sm="3" md="2" class="firstCol">
+            <label><strong>Registered Office</strong></label>
+          </v-col>
+          <v-col cols="10" sm="9" md="10" class="secondCol">
+            {{getBusinessContact.email || '(Not entered)'}}
+          </v-col>
+        </v-row>
+      </v-container>
+      <v-container v-if="isRoleStaff">
+        <v-row class="pl-4">
+          <v-col cols="2" sm="3" md="2" class="firstCol">
+            <label><strong>User Account</strong></label>
+          </v-col>
+          <v-col cols="10" sm="9" md="10" class="secondCol">
+            {{ getUserEmail }}
+          </v-col>
+        </v-row>
+      </v-container>
+      <v-container v-else>
+        <v-row class="pl-4">
+          <v-col cols="2" sm="3" md="2" class="firstCol">
+            <label><strong>Optional Email</strong></label>
+          </v-col>
+          <v-col cols="10" sm="9" md="10" class="secondCol">
+            <v-text-field
+            v-model="optionalEmail"
+            id="optionalEmail"
+            class="text-input-field"
+            filled
+            label="Optional Email"
+            hint="Example: name@email.com"
+            persistent-hint
+            :rules="entityEmailRules"
+          >
+          </v-text-field>
+          </v-col>
+        </v-row>
+      </v-container>
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Emit, Prop, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { CommonMixin } from '@/mixins'
+import { ActionBindingIF } from '@/interfaces'
+// Shared Interfaces
+import { ContactPointIF } from '@bcrs-shared-components/interfaces'
+
+@Component({})
+export default class DocumentsDelivery extends Mixins(CommonMixin) {
```

Comment:
Does this component have a valid/invalid state that it needs to emit to its parent?

Also, if yes, please add some statements to the unit test to verify that valid/invalid are emitted accordingly.

If not... never mind :)